src/libsystemd/sd-journal/journal-authenticate.c

```diff
@@ -76,7 +78,11 @@ int journal_file_hmac_start(JournalFile *f) {
         /* Prepare HMAC for next cycle */
         gcry_md_reset(f->hmac);
         FSPRG_GetKey(f->fsprg_state, key, sizeof(key), 0);
-        gcry_md_setkey(f->hmac, key, sizeof(key));
+        err = gcry_md_setkey(f->hmac, key, sizeof(key));
+        if (gcry_err_code(err))
```

Comment:
Shouldn't it be `!=` GPG_ERR_NO_ERROR`?

Reply:
documentation explicitly says it's 0, but beng extra careful sounds good